test/test_nn.py

```diff
@@ -8956,6 +8956,37 @@ def __call__(self, input):
     input_size=(4, 16),
     fullname='AdaptiveLogSoftmax'))
 
+class _LayerNormWithNonDefaultWeight(nn.LayerNorm):
```

Comment:
Why do you need a brand new class here? Can't you just update the weights from the original class after creation?

Reply:
Removed.